### PR TITLE
Add django-guardian

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

File: recipes/django-guardian/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```
Comment (synapticarbors) [inline recipes/django-guardian/meta.yaml]: You need to add `pytest-runner` as a `build` requirement since it's listed in the `setup_requires`:

https://github.com/django-guardian/django-guardian/blob/devel/setup.py#L15

Comment (sannykr) [inline recipes/django-guardian/meta.yaml]: Thanks. I added the pytest-runner, but my circleci check still failed. Is there anything else I need to add? Should I add 'mock', 'django-environ', 'pytest', 'pytest-django' to the run-test.py file?

Comment (synapticarbors) [inline recipes/django-guardian/meta.yaml]: It looks like CircleCI might be having system issues that are unrelated to your particular recipe.

Comment (sannykr) [inline recipes/django-guardian/meta.yaml]: Now, it has passed all checks. Could you please merge it.
